### ISSUE TITLE
Code review feature

### ISSUE THREAD

**Issue by lafriks** [go-gitea/gitea#14745]
Would be nice to have feature to create Code review tasks. With comments on code like in PR but for specific repository branch.

**Comment by lunny** [comment]
Isn't this equal to #4898 ?

**Comment by lafriks** [comment]
Not really I want more like PR reviews but for whole branch not specific commit that would be marked outdated if changes pushed to to that branch

**Comment by lunny** [comment]
So that is also a commit comment or review but with an outdate for old commit of the branch automatically.

**Comment by bircni** [comment]
this is already solved via using PullRequests?

**Comment by lafriks** [comment]
Not really, this is a bit different, my use case is example to have security audit or code review on whole default branch not on single pull request
